src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -114,100 +137,244 @@ object VCFOutput {
 
     def allelicDepthString(depths: Map[String, Int]): String = {
       val allAlleles = (Seq(allele.ref, allele.alt) ++ depths.keys.toSeq).distinct
-      allAlleles.map(x => "%s=%d".format(if (x.isEmpty) "." else x, depths.getOrElse(x, 0))).mkString(",")
+      allAlleles.map(
```

Comment:
I find this increase in verbosity less clear actually. I'd much rather be able to see the code on one page then have to scroll around a ton. That's a personal preference though and I wouldn't reject new code that looks like this. I just don't think we should change existing code to be in this style, as it is not a meaningful improvement in my opinion